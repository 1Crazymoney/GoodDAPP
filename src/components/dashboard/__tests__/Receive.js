import React from 'react'
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

import { getWebRouterComponentWithMocks } from './__util__'

describe('Receive', () => {
  it('renders without errors', () => {
    const Receive = getWebRouterComponentWithMocks('../Receive')
    const tree = renderer.create(<Receive />)
    expect(tree.toJSON()).toBeTruthy()
  })

  // it('matches snapshot', () => {
  //   const Receive = getWebRouterComponentWithMocks('../Receive')
  //   const component = renderer.create(<Receive />)
  //   const tree = component.toJSON()
  //   expect(tree).toMatchSnapshot()
  // })
})
