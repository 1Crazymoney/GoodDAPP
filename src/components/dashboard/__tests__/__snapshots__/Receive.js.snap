// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Receive matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-position-bnwqim"
    data-name="viewWrapper"
    style={
      Object {
        "WebkitBoxDirection": "normal",
        "WebkitBoxFlex": 1,
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "WebkitFlexGrow": 1,
        "backgroundImage": "linear-gradient(3.141592653589793rad,#00AFFF,#2DC0F7,#28C0EF,#23C0E7,#1EC1DF,#19C1D7,#14C1CF,#0FC2C7,#0FC2C7,#0AC2BF,#05C2B7,#00C3AF)",
        "display": "flex",
        "flexDirection": "column",
        "flexGrow": 1,
        "maxHeight": "844px",
        "msFlexDirection": "column",
        "msFlexPositive": 1,
        "paddingBottom": "8px",
        "paddingLeft": "8px",
        "paddingRight": "8px",
        "paddingTop": "8px",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n r-height-yivrw8 r-justifyContent-1777fci r-marginBottom-5oul0u r-paddingBottom-xd6kpl r-paddingLeft-f727ji r-paddingRight-1pyaxff r-paddingTop-tskmnb"
      style={
        Object {
          "backgroundColor": "rgba(255,255,255,1.00)",
          "borderBottomLeftRadius": "5px",
          "borderBottomRightRadius": "5px",
          "borderTopLeftRadius": "5px",
          "borderTopRightRadius": "5px",
          "paddingBottom": "16px",
          "paddingLeft": "12px",
          "paddingRight": "12px",
          "paddingTop": "16px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
        style={
          Object {
            "WebkitBoxPack": "justify",
            "WebkitJustifyContent": "space-between",
            "justifyContent": "space-between",
            "msFlexPack": "justify",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexDirection-18u37iz"
          style={
            Object {
              "WebkitAlignItems": "baseline",
              "WebkitBoxAlign": "baseline",
              "alignItems": "baseline",
              "display": "flex",
              "msFlexAlign": "baseline",
            }
          }
        >
          <div
            className="css-text-901oao"
            dir="auto"
            style={
              Object {
                "color": "rgba(85,85,85,1.00)",
                "fontFamily": "Roboto Slab",
                "fontSize": "36px",
                "fontWeight": "700",
                "lineHeight": "36px",
                "marginRight": "2px",
                "textAlign": "right",
                "textDecoration": "none",
                "textTransform": "none",
              }
            }
          >
            -.--
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
            style={
              Object {
                "WebkitBoxPack": "justify",
                "WebkitJustifyContent": "space-between",
                "justifyContent": "space-between",
                "msFlexPack": "justify",
              }
            }
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(85,85,85,1.00)",
                  "fontFamily": "Roboto Slab",
                  "fontSize": "18px",
                  "fontWeight": "700",
                  "lineHeight": "22px",
                  "textAlign": "right",
                  "textDecoration": "none",
                  "textTransform": "none",
                }
              }
            >
              G
            </div>
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(85,85,85,1.00)",
                  "fontFamily": "Roboto Slab",
                  "fontSize": "17.5px",
                  "fontWeight": "700",
                  "lineHeight": "22px",
                  "textAlign": "right",
                  "textDecoration": "none",
                  "textTransform": "none",
                }
              }
            >
              $
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n"
        />
      </div>
    </div>
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxFlex": 1,
          "WebkitFlexGrow": 1,
          "backgroundColor": "rgba(255,255,255,1.00)",
          "borderBottomLeftRadius": "5px",
          "borderBottomRightRadius": "5px",
          "borderTopLeftRadius": "5px",
          "borderTopRightRadius": "5px",
          "flexGrow": 1,
          "msFlexPositive": 1,
          "paddingBottom": "16px",
          "paddingLeft": "12px",
          "paddingRight": "12px",
          "paddingTop": "16px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "height": "25%",
          }
        }
      />
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-justifyContent-1777fci"
        style={
          Object {
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "vertical",
            "WebkitFlexDirection": "column",
            "flexDirection": "column",
            "msFlexDirection": "column",
          }
        }
      >
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(66,69,74,1.00)",
              "fontFamily": "Roboto",
              "fontSize": "16px",
              "fontWeight": "500",
              "lineHeight": "22px",
              "marginBottom": "24px",
              "textAlign": "center",
              "textDecoration": "none",
              "textTransform": "none",
            }
          }
        >
          Let someone scan your wallet address
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-justifyContent-1777fci"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "borderBottomColor": "rgba(0,175,255,1.00)",
                "borderBottomLeftRadius": "5px",
                "borderBottomRightRadius": "5px",
                "borderBottomWidth": "1px",
                "borderLeftColor": "rgba(0,175,255,1.00)",
                "borderLeftWidth": "1px",
                "borderRightColor": "rgba(0,175,255,1.00)",
                "borderRightWidth": "1px",
                "borderTopColor": "rgba(0,175,255,1.00)",
                "borderTopLeftRadius": "5px",
                "borderTopRightRadius": "5px",
                "borderTopWidth": "1px",
                "paddingBottom": "16px",
                "paddingLeft": "16px",
                "paddingRight": "16px",
                "paddingTop": "16px",
              }
            }
          >
            <canvas
              height={128}
              style={
                Object {
                  "height": 128,
                  "width": 128,
                }
              }
              theme={
                Object {
                  "animation": Object {
                    "scale": 1,
                  },
                  "backdrop": Object {
                    "backgroundColor": "rgba(255, 0, 0, 0.8)",
                  },
                  "borders": Object {
                    "defaultBorderColor": "#c9c8c9",
                  },
                  "colors": Object {
                    "accent": "#03dac4",
                    "backdrop": "rgba(0, 0, 0, 0.5)",
                    "background": "#f6f6f6",
                    "blue": "#006EA0",
                    "darkBlue": "#0C263D",
                    "darkGray": "#42454A",
                    "disabled": "#E3E3E2",
                    "error": "#FA6C77",
                    "facebookBlue": "#1877F3",
                    "googleBlue": "#3367D6",
                    "googleRed": "#EA4335",
                    "gray": "#555",
                    "gray50Percent": "#CBCBCB",
                    "gray80Percent": "#A3A3A3",
                    "green": "#00C3AE",
                    "lightGray": "#EEE",
                    "lightGreen": "#00BBA5",
                    "lighterGray": "#A3A3A3",
                    "notification": "#f50057",
                    "onBackground": "#000000",
                    "onSurface": "#000000",
                    "orange": "#F8AF40",
                    "placeholder": "#CBCBCB",
                    "primary": "#00AFFF",
                    "purple": "#9F6A9D",
                    "red": "#FA6C77",
                    "surface": "#ffffff",
                    "text": "#222",
                    "white": "#ffffff",
                  },
                  "dark": false,
                  "feedItems": Object {
                    "borderRadius": 8,
                    "height": 84,
                    "itemBackgroundColor": "#fff",
                  },
                  "fontStyle": Object {
                    "color": "#555",
                    "fontSize": 18,
                    "textAlign": "center",
                  },
                  "fonts": Object {
                    "default": "Roboto",
                    "light": Object {
                      "fontFamily": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                      "fontWeight": "300",
                    },
                    "medium": Object {
                      "fontFamily": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                      "fontWeight": "500",
                    },
                    "regular": Object {
                      "fontFamily": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                      "fontWeight": "400",
                    },
                    "slab": "Roboto Slab",
                    "thin": Object {
                      "fontFamily": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                      "fontWeight": "100",
                    },
                  },
                  "modals": Object {
                    "activityIndicatorBackgroundColor": "rgba(255, 255, 255, 0.8)",
                    "backgroundColor": "#fff",
                    "borderLeftWidth": 10,
                    "borderRadius": 5,
                    "contentPadding": 16,
                    "jaggedEdgeSize": 16,
                    "overlayBackgroundColor": "transparent",
                    "overlayHorizontalPadding": 20,
                    "overlayVerticalPadding": 65,
                  },
                  "paddings": Object {
                    "defaultMargin": 8,
                    "mainContainerPadding": 8,
                  },
                  "roundness": 4,
                  "sizes": Object {
                    "borderRadius": 5,
                    "default": 8,
                    "defaultDouble": 16,
                    "defaultHalf": 4,
                    "defaultQuadruple": 32,
                    "maxHeightForTabletAndDesktop": 844,
                    "maxWidthForTabletAndDesktop": 475,
                    "minHeightForDialogMessage": 350,
                  },
                }
              }
              width={128}
            />
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-justifyContent-1777fci"
        style={
          Object {
            "WebkitBoxDirection": "normal",
            "WebkitBoxFlex": 1,
            "WebkitBoxOrient": "vertical",
            "WebkitFlexDirection": "column",
            "WebkitFlexGrow": 1,
            "flexDirection": "column",
            "flexGrow": 1,
            "marginBottom": "20px",
            "marginTop": "20px",
            "msFlexDirection": "column",
            "msFlexPositive": 1,
          }
        }
      >
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(66,69,74,1.00)",
              "fontFamily": "Roboto",
              "fontSize": "14px",
              "fontWeight": "normal",
              "lineHeight": "20px",
              "textAlign": "center",
              "textDecoration": "none",
              "textTransform": "none",
            }
          }
        >
          - OR -
        </div>
      </div>
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "stretch",
            "WebkitBoxAlign": "stretch",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "vertical",
            "WebkitBoxPack": "justify",
            "WebkitFlexDirection": "column",
            "WebkitJustifyContent": "space-between",
            "alignItems": "stretch",
            "flexDirection": "column",
            "justifyContent": "space-between",
            "msFlexAlign": "stretch",
            "msFlexDirection": "column",
            "msFlexPack": "justify",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-justifyContent-1777fci"
          style={
            Object {
              "WebkitAlignItems": "stretch",
              "WebkitBoxAlign": "stretch",
              "alignItems": "stretch",
              "backgroundColor": "rgba(0,0,0,0.00)",
              "borderBottomColor": "rgba(0,175,255,1.00)",
              "borderBottomLeftRadius": "50px",
              "borderBottomRightRadius": "50px",
              "borderBottomStyle": "solid",
              "borderBottomWidth": "1px",
              "borderLeftColor": "rgba(0,175,255,1.00)",
              "borderLeftStyle": "solid",
              "borderLeftWidth": "1px",
              "borderRightColor": "rgba(0,175,255,1.00)",
              "borderRightStyle": "solid",
              "borderRightWidth": "1px",
              "borderTopColor": "rgba(0,175,255,1.00)",
              "borderTopLeftRadius": "50px",
              "borderTopRightRadius": "50px",
              "borderTopStyle": "solid",
              "borderTopWidth": "1px",
              "display": "flex",
              "marginBottom": "0px",
              "marginLeft": "0px",
              "marginRight": "0px",
              "marginTop": "0px",
              "minHeight": "44px",
              "minWidth": "64px",
              "msFlexAlign": "stretch",
              "paddingBottom": "0px",
              "paddingLeft": "0px",
              "paddingRight": "0px",
              "paddingTop": "0px",
            }
          }
        >
          <div
            className="css-cursor-18t94o4 css-view-1dbjc4n r-cursor-1loqt21 r-flex-13awgt0 r-overflow-1udh08x r-position-bnwqim r-touchAction-1otgn73"
            data-focusable={true}
            disabled={false}
            onKeyDown={[Function]}
            onKeyPress={[Function]}
            onKeyUp={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            role="button"
            style={
              Object {
                "WebkitAlignSelf": "stretch",
                "alignSelf": "stretch",
                "borderBottomLeftRadius": "50px",
                "borderBottomRightRadius": "50px",
                "borderTopLeftRadius": "50px",
                "borderTopRightRadius": "50px",
                "msFlexItemAlign": "stretch",
              }
            }
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-18u37iz r-justifyContent-1777fci"
              style={
                Object {
                  "WebkitAlignSelf": "stretch",
                  "alignSelf": "stretch",
                  "msFlexItemAlign": "stretch",
                }
              }
            >
              <div
                className="css-view-1dbjc4n r-marginHorizontal-1ydw1k6 r-marginVertical-712mwr"
              >
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-18u37iz"
                  style={
                    Object {
                      "display": "flex",
                    }
                  }
                >
                  <div
                    className="css-text-901oao r-alignItems-1awozwy r-justifyContent-1777fci"
                    dir="auto"
                    style={
                      Object {
                        "color": "rgba(0,175,255,1.00)",
                        "fontFamily": "Roboto",
                        "fontSize": "16px",
                        "fontWeight": "500",
                        "letterSpacing": "0px",
                        "lineHeight": "22px",
                        "paddingTop": "1px",
                        "textAlign": "center",
                        "textDecoration": "none",
                        "textTransform": "uppercase",
                      }
                    }
                  >
                    Request specific amount
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "height": "16px",
            }
          }
        />
        <div
          className="css-view-1dbjc4n r-justifyContent-1777fci"
          data-gdtype="copybutton"
          style={
            Object {
              "WebkitAlignItems": "stretch",
              "WebkitBoxAlign": "stretch",
              "alignItems": "stretch",
              "backgroundColor": "rgba(0,175,255,1.00)",
              "borderBottomColor": "rgba(0,175,255,1.00)",
              "borderBottomLeftRadius": "50px",
              "borderBottomRightRadius": "50px",
              "borderBottomStyle": "solid",
              "borderBottomWidth": "0px",
              "borderLeftColor": "rgba(0,175,255,1.00)",
              "borderLeftStyle": "solid",
              "borderLeftWidth": "0px",
              "borderRightColor": "rgba(0,175,255,1.00)",
              "borderRightStyle": "solid",
              "borderRightWidth": "0px",
              "borderTopColor": "rgba(0,175,255,1.00)",
              "borderTopLeftRadius": "50px",
              "borderTopRightRadius": "50px",
              "borderTopStyle": "solid",
              "borderTopWidth": "0px",
              "boxShadow": "0px 0.75px 1.5px rgba(0,0,0,0.24)",
              "display": "flex",
              "marginBottom": "0px",
              "marginLeft": "0px",
              "marginRight": "0px",
              "marginTop": "0px",
              "minHeight": "44px",
              "minWidth": "64px",
              "msFlexAlign": "stretch",
              "paddingBottom": "0px",
              "paddingLeft": "0px",
              "paddingRight": "0px",
              "paddingTop": "0px",
            }
          }
        >
          <div
            className="css-cursor-18t94o4 css-view-1dbjc4n r-cursor-1loqt21 r-flex-13awgt0 r-overflow-1udh08x r-position-bnwqim r-touchAction-1otgn73"
            data-focusable={true}
            disabled={false}
            onKeyDown={[Function]}
            onKeyPress={[Function]}
            onKeyUp={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            role="button"
            style={
              Object {
                "WebkitAlignSelf": "stretch",
                "alignSelf": "stretch",
                "borderBottomLeftRadius": "50px",
                "borderBottomRightRadius": "50px",
                "borderTopLeftRadius": "50px",
                "borderTopRightRadius": "50px",
                "msFlexItemAlign": "stretch",
              }
            }
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-18u37iz r-justifyContent-1777fci"
              style={
                Object {
                  "WebkitAlignSelf": "stretch",
                  "alignSelf": "stretch",
                  "msFlexItemAlign": "stretch",
                }
              }
            >
              <div
                className="css-view-1dbjc4n r-marginHorizontal-1ydw1k6 r-marginVertical-712mwr"
              >
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-18u37iz"
                  style={
                    Object {
                      "display": "flex",
                    }
                  }
                >
                  <div
                    className="css-text-901oao r-alignItems-1awozwy r-justifyContent-1777fci"
                    dir="auto"
                    style={
                      Object {
                        "color": "rgba(255,255,255,1.00)",
                        "fontFamily": "Roboto",
                        "fontSize": "16px",
                        "fontWeight": "500",
                        "letterSpacing": "0px",
                        "lineHeight": "22px",
                        "paddingTop": "1px",
                        "textAlign": "center",
                        "textDecoration": "none",
                        "textTransform": "uppercase",
                      }
                    }
                  >
                    Receive via wallet link
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
